hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFileReader.java

```diff
@@ -287,7 +288,6 @@ private boolean isBlockCorrupted(int blocksize) throws IOException {
       // This seek is required because contract of seek() is different for naked DFSInputStream vs BufferedFSInputStream
       // release-3.1.0-RC1/DFSInputStream.java#L1455
       // release-3.1.0-RC1/BufferedFSInputStream.java#L73
-      inputStream.seek(currentPos);
```

Comment:
We need to reinstate this one

Reply:
oops. I missed to push out my local change. will fix it.